### DISCUSSION TITLE
Major macOS upgrade

### DISCUSSION THREAD

**Discussion by n-schoofs** [usnistgov/macos_security#287] [Q&A] [answered]
Hi there

Any best practices on how to handle major macOS upgrades in combination with Jamf Pro and the mSCP project?
E.g., the upgrade from macOS 12 to macOS 13 famously introduced System Settings resulting in failed compliance results because of the .plist file still containing results for System Preferences. Thank you for your input/ideas on this :-)

**Comment by jmahlman** [comment, marked as answer]
In my environment, I have a smart group that checks for `os version greater than 13.0` and I check my extension attribute that lists any findings for `sysprefs`. This smart group will run the remediation with the `--reset` flag.

**Reply by n-schoofs** [reply]
Makes sense, thanks!
